tests/test_api_calls.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+import pytest
+# add src to path to be able to run the tests
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__name__), 'src')))
+from utils.api_calls import get_location, get_request_co2signal
+
+
+def test_get_location():
+    # Get location
+    geolocation = get_location()
+    # Check that the dictionary is not empty
+    assert geolocation
+    # Check that the dictionary contains the expected keys
+    assert 'longitude' in geolocation
+    assert 'latitude' in geolocation
+    # Check that the dictionary contains the expected values
+    assert isinstance(geolocation['longitude'], str)
+    assert isinstance(geolocation['latitude'], str)
+    assert float(geolocation['latitude'])
+    assert float(geolocation['longitude'])
```

Comment:
Isn't it enough to keep these last 4 tests and remove the previous ones? They implicitly test for the others as well.

Reply:
given that this testing is not interactive, by keeping them separate you will spot more easily the cause of the error. if you go the actions tab you can see examples of the output of the test and have a feeling